### ISSUE TITLE
How can I set training without validation?

### ISSUE THREAD

**Issue by tzjtatata** [volcengine/verl#1676]
I want to train a model without any validation. But I found that "must validation" is hard-coding in verl. HOW CAN I AVOID IT?

**Comment by vermouth1992** [comment]
You can set the `test_freq` to a huge value

**Comment by tzjtatata** [comment]
I don't have val dataset. But verl force me to put one...



---Original---
From: "Chi ***@***.***&gt;
Date: Sun, May 25, 2025 16:30 PM
To: ***@***.***&gt;;
Cc: "Yuanze ***@***.******@***.***&gt;;
Subject: Re: [volcengine/verl] How can I set training without validation?(Issue #1676)


vermouth1992 left a comment (volcengine/verl#1676)
 
You can set the test_freq to a huge value

**Comment by ccclyu** [comment]
simply copy some samples from the training set as the validation set. It is weird to discard the validation stage in terms of the functionality of training framework.

**Comment by HareshKarnan** [comment]
training seems to hang at the step iteration that matches test_freq, makes me wonder if there's a bug causing training to hang at the validation stage
